sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -953,8 +955,16 @@ public ServiceBusSenderAsyncClient buildAsyncClient() {
                         new IllegalArgumentException("Unknown entity type: " + entityType));
             }
 
+            String clientIdentifier;
```

Comment:
nit: Since we're using an if/else, I think this can be `final`.

Reply:
Same with the other instance below. I like `final` because you get compiler errors if you try to reassign it to make sure there's no unintentional reassignment. Personal preference tho!

Reply:
Sure! will change it!